components/batch-system/src/router.rs

```diff
@@ -36,8 +36,8 @@ pub struct Router<N: Fsm, C: Fsm, Ns, Cs> {
     pub(crate) normal_scheduler: Ns,
     control_scheduler: Cs,
 
-    // Indicates the router is shutted down or not.
-    shutted: Arc<AtomicBool>,
+    // Indicates the router is shut down or not.
+    shut: Arc<AtomicBool>,
```

Comment:
I don't think `shut` is equals to `shut down`. Maybe `shutdown` is better and clear.